core-ui/src/components/Preferences/ThemeChooser.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useTheme, Dropdown } from 'react-shared';
+import { LayoutPanel } from 'fundamental-react';
+
+const THEMES = [
+  {
+    key: 'default',
+    text: 'Light (default)',
+  },
+  // {
```

Comment:
Can it be removed?

Reply:
I left it on purpose for the reviewer to uncomment the code, check this theme out and confirm or deny my finding that this is identical to the `dark` theme.

Reply:
For me it is identical as light theme. Looks like something is loading properly